pkg/webui/components/dropdown/index.js

```diff
@@ -46,15 +46,18 @@ Dropdown.defaultProps = {
   onItemsClick: () => null,
 }
 
-const DropdownItem = function({ icon, title, path, action, exact }) {
+const DropdownItem = ({ icon, title, path, action, exact, showActive }) => {
   const iconElement = icon && <Icon className={style.icon} icon={icon} nudgeUp />
+  const activeClassName = classnames({
+    [style.active]: showActive,
+  })
   const ItemElement = action ? (
     <button onClick={action} onKeyPress={action} role="tab" tabIndex="0">
       {iconElement}
       <Message content={title} />
     </button>
   ) : (
-    <NavLink activeClassName={style.active} to={path} exact={exact}>
+    <NavLink activeClassName={activeClassName} to={path} exact={exact}>
```

Comment:
```suggestion
    <NavLink activeClassName={activeClassName} to={path} exact={exact} tabIndex={tabIndex}>
```

You have to pass the `tabIndex` prop here, and in the `button` element above. See my comment below.